IBPSA/Airflow/Multizone/BaseClasses/TwoWayFlowElement.mo

```diff
@@ -41,9 +41,9 @@ protected
   Modelica.Units.SI.VolumeFlowRate VZer_flow(fixed=false)
     "Minimum net volume flow rate to prevent zero flow";
 
-  Modelica.Units.SI.Mass mExcAB(start=0, fixed=true)
+  output Modelica.Units.SI.Mass mExcAB(start=0, fixed=true)
     "Air mass exchanged (for purpose of error control only)";
-  Modelica.Units.SI.Mass mExcBA(start=0, fixed=true)
+  output Modelica.Units.SI.Mass mExcBA(start=0, fixed=true)
```

Comment:
Why is an `output` qualifier needed for a state variable?

Reply:
Togheter with the next change, the ouput qualifier helps the IDE interpret the causality of the equations. Here specicially it might not be needed but it is better to do it if the causality is known.